docker-compose.yaml

```diff
@@ -33,22 +33,22 @@ services:
   streaming-binlog: # To run independently: docker-compose up --build streaming-binlog
     build:
       context: streaming-binlog
-    image: openmrs-fhir-analytics/streaming-pipeline:${DEBEZIUM_VERSION}
+    image: openmrs-fhir-analytics/streaming-binlog:latest
     container_name: streaming-pipeline
     network_mode: host
     healthcheck:
       test: "exit 0"
     volumes:
-      - ./dbz:/tmp
-      - ./utils/dbz_event_to_fhir_config.json:/deployments/utils/dbz_event_to_fhir_config.json
+      - ./utils/dbz_event_to_fhir_config.json:/deployments/dbz_event_to_fhir_config.json
+      # parquet_store is the directory which you want to persist the generated parquet files
+      - ./parquet_store:/tmp
```

Comment:
Suggestion: Either add this to `.gitignore` or move this under `/data/`.

Reply:
sure!

Reply:
I have switched to using  /data/